cli/enry/main.go

```diff
@@ -32,24 +32,30 @@ func main() {
 		log.Fatal(err)
 	}
 
-	errors := false
+	fileInfo, err := os.Stat(root)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	if fileInfo.Mode().IsRegular() {
+		printFileAnalysis(root)
+		os.Exit(0)
+	}
+
 	out := make(map[string][]string, 0)
 	err = filepath.Walk(root, func(path string, f os.FileInfo, err error) error {
 		if err != nil {
-			errors = true
```

Comment:
These lines were dead code, it were used before to handle an status code on exit, but it was removed leaving this useless `errors` variable